shell/platform/windows/BUILD.gn

```diff
@@ -100,6 +101,63 @@ shared_library("flutter_windows") {
   public_configs = [ "$flutter_root:config" ]
 }
 
+test_fixtures("flutter_windows_fixtures") {
+  fixtures = []
+}
+
+executable("flutter_windows_unittests") {
+  testonly = true
+
+  sources = [
+    "win32_window_unittests.cc",
+    "testing/win32_flutter_window_test.h",
+    "testing/win32_flutter_window_test.cc",
+    "angle_surface_manager.cc",
+    "angle_surface_manager.h",
+    "flutter_windows.cc",
+    "key_event_handler.cc",
+    "key_event_handler.h",
+    "keyboard_hook_handler.h",
+    "platform_handler.cc",
+    "platform_handler.h",
+    "text_input_plugin.cc",
+    "text_input_plugin.h",
+    "win32_dpi_helper.cc",
+    "win32_dpi_helper.h",
+    "win32_flutter_window.cc",
+    "win32_flutter_window.h",
+    "win32_window.cc",
+    "win32_window.h",
+    "window_state.h",
```

Comment:
Why is this duplicating everything instead of depending on `flutter_windows_source`?